project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/states/_activity_global.scss

```diff
@@ -63,3 +63,67 @@
 .message-c4m-insert--node--share {
   margin-top: 25px;
 }
+
+body:not(.no-js) {
+  .field-name-activity-stream-by-topic {
```

Comment:
Not sure why do we need all this CSS rules, probably using the `pane-global-activity-stream-no-js-fallback` and `pane-activity-stream` would be enough ...